kitsune/wiki/views.py

```diff
@@ -207,6 +210,17 @@ def document(request, document_slug, document=None):
     # The list above was built backwards, so flip this.
     breadcrumbs.reverse()
 
+    # Decide whether the sumo banner should be displayed in the KB.
+    # Base the decision on the English version of the document. If this is True
+    # all translation should display the banner
+    if doc.locale != settings.WIKI_DEFAULT_LANGUAGE:
+        doc_for_banner = doc.parent if doc.parent.locale == settings.WIKI_DEFAULT_LANGUAGE else None
```

Comment:
If the parent document isn't in `settings.WIKI_DEFAULT_LANGUAGE` this will blow up the `re.search` below. Rather than `else None` this should be `else ""`

Reply:
Also this line is too long: https://circleci.com/gh/mozilla/kitsune/2605?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link